### PR TITLE
Add label by Jira issue type if it is exists

### PR DESCRIPTION
**Pull request by uchqunbek**

### CODE REVIEW

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

**Comment by doniyor2109** [comment]
> @doniyor2109 how I can fix the failed PR check?

Fixed checks

File: jira/add-project-label/index.ts [hunk 1]
```diff
@@ -44,16 +44,39 @@ async function main() {
...
+  );
+
   const hasPRLabel = pr.labels.find((x) => x.name === projectLabel);
 
   if (!hasPRLabel) {
+    let labelsToAdd = [projectLabel];
+    if (hasIssueTypeLabelExists && !hasIssueTypeLabelExistsInPR) {
+      const labelName = labelMap.get(issueTypeName);
+      if (labelName) {
+        labelsToAdd.push(labelName);
```
[the remark this answers is not in the document]
Comment (uchqunbek) [reply jira/add-project-label/index.ts:70]: @doniyor2109 I wanted to add a label only if it exists in the repository. So in this case any other repositories without existing labels won't be affected. Do you think creating a label is a better choice?

[the remark this answers is not in the document]
Comment (uchqunbek) [reply jira/add-project-label/index.ts]: @doniyor2109 Updated

File: jira/add-project-label/index.ts [hunk 2]
```diff
@@ -44,16 +45,43 @@ async function main() {
...
+    }
+  }
+
   const hasPRLabel = pr.labels.find((x) => x.name === projectLabel);
 
   if (!hasPRLabel) {
+    let labelsToAdd = [projectLabel];
+    if (issueLabelToAdd !== undefined) {
+      labelsToAdd.push(issueLabelToAdd);
+    }
```
Comment (doniyor2109) [inline jira/add-project-label/index.ts:77]: I think we should check for issueLabel as well in if condition

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**